src/components/layout/Workflow/Workflow.tsx

```diff
@@ -1,10 +1,31 @@
 import React from 'react';
 
 import { useGlobalState } from '../../../state/contexts/GlobalState';
-import { WorkflowProps } from '../../../types';
 import WorkflowButtons from '../../inputs/buttons/WorkflowButtons/WorkflowButtons';
 
-function Workflow({ stages, onNext, onBack, stage }: WorkflowProps) {
+function Workflow({
+  stages,
+  onNext,
+  onBack,
+  stage,
+}: {
+  stage: number;
+  onNext: () => void;
+  onBack: () => void;
+  stages: {
+    [x: number]: {
+      component: JSX.Element;
+      showNext?: boolean;
+      showBack?: boolean;
+      disableNext?: boolean;
+      requiresWallet?: boolean;
+      customNextStyle?: any;
+      customBackStyle?: any;
+      backText?: string;
+      nextText?: string;
```

Comment:
no problem with this change, we can move component specific types to the component `.tsx` file

```
type WorkflowProp {
..
}

function Workflow({
...: WorkflowProp
) => 
}

Reply:
added